lib/Sema/BuilderTransform.cpp

```diff
@@ -1387,6 +1387,63 @@ BraceStmt *swift::applyFunctionBuilderTransform(
         captured.first, captured.second)));
 }
 
+/// Produce any additional syntactic diagnostics for the body of a
```

Comment:
Looks like this doc comment is incomplete :)

Reply:
Fixed locally, thanks! I'm going to let CI run a bit before pushing the change.